pkg/controller.v1/tensorflow/controller.go

```diff
@@ -273,13 +273,13 @@ func (tc *TFController) processNextWorkItem() bool {
 	}
 
 	// Wait until queuing annotation is removed
-	if tfJob.Annotations != nil {
-		if suspend, exist := tfJob.Annotations[suspendInQueue]; exist && suspend == "true" {
-			infoMsg := fmt.Sprintf("Annotation %s is found, operator will not process until removed", suspendInQueue)
-			tflogger.LoggerForKey(key).Info(infoMsg)
-			return true
-		}
-	}
+	// if tfJob.Annotations != nil {
```

Comment:
Why not using `isSuspend` to avoid creating resources like pod? If it's for ECI scenario,  will it create pod and delete it after a while?